book/versioned_docs/version-4.0.0/developers/common-issues.md

```diff
@@ -139,3 +141,15 @@ The Rust compiler had breaking changes to its names of available options between
 This message indicates that you're trying to use `sp1-build` < `4.0.0` with the 1.82 toolchain,
 `sp1-build` versions >= 4.0.0 have support for the 1.82 and 1.81 toolchains.
 
+## Ubuntu 20.04
+
+The Succinct Rust toolchain is built on Ubuntu 22.04, which links a newer version of libc than available
+on Ubuntu 20.04.
+
+This error may manifest as:
+```bash
+   /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found 
+```
+
+This most commonly happens in CI runners, it is recommended to bump the runner OS to 22.04,
```

Comment:
This most commonly occurs in CI runners. Increasing the runner OS to 22.04 solves the issue.

This issue also applies to all older Linux distributions, as the Rust toolchain was built on a machine with `20.04` Ubuntu on it.